featurebyte/service/feature_store_warehouse.py

```diff
@@ -367,7 +371,14 @@ async def table_preview(self, location: TabularSource, limit: int) -> dict[str,
         db_session = await self.session_manager_service.get_feature_store_session(
             feature_store=feature_store, timeout=INTERACTIVE_SESSION_TIMEOUT_SECONDS
         )
-        sql_expr = get_source_expr(source=location.table_details).limit(limit)
+        sql_expr = get_source_expr(source=location.table_details)
+
+        # apply order by if specified
+        if order_by_column:
+            sql_expr = sql_expr.order_by(order_by_column)
```

Comment:
It would be a more robust to wrap the column in `quoted_identifier` so that it can work with any column names even those with special characters.